data-dashboard/server/repositories/board.ts

```diff
@@ -22,4 +23,19 @@ export class PrismaBoardRepository implements IBoardRepository {
       title: board.title,
     }));
   }
+
+  async findOne(id: string): Promise<Board> {
+    try {
+      const board = await this.db.boards.findFirstOrThrow({ where: { board_id: id } });
+      return {
+        boardId: board.board_id,
+        createdAt: new Date(board.createdAt),
+        name: board.title,
```

Comment:
I removed "name" in PR how are we going to handle this?

Reply:
We will need to update your PR to include this change, otherwise the build will fail.